sencha-workspace/SlateAdmin/app/controller/People.js

```diff
@@ -144,7 +145,8 @@ Ext.define('SlateAdmin.controller.People', {
                 click: me.onSearchClick
             },
             'people-navpanel #groups': {
-                select: me.onGroupSelect
+                //select: me.onGroupSelect
+                itemclick: me.onGroupSelect
```

Comment:
@BillClinton why switch from select to itemclick? I usually prefer select because it will only fire when a new thing is selected rather than repeatedly if you click on the same thing. Also I think select will fire if you move the selection with the arrows keys whereas itemclick will not. Did you run into some problem using select?